### PR TITLE
Change - ## Delegate to a Validator - part

### PR DESCRIPTION
**Pull request by JB273**
This part is still using Devnet commands, changed it to reflect mamaki values.

### CODE REVIEW

File: docs/nodes/mamaki-testnet.md [hunk 1]
```diff
@@ -120,12 +120,10 @@ You can return back to where you left off in the Bridge Node guide [here](./vali
...
 
-To delegate tokens to the the `celesvaloper` validator, as an example you can run:
+To delegate tokens to the the `celestiavaloper` validator, as an example you can run:
 
-```sh
-celestia-appd tx staking delegate \
-    celesvaloper1q3v5cugc8cdpud87u4zwy0a74uxkk6u43cv6hd 1000000utia \
-    --from=$VALIDATOR_WALLET --chain-id=mamaki
+```
+celestia-appd tx staking delegate celesvaloper1q3v5cugc8cdpud87u4zwy0a74uxkk6u43cv6hd 1000000utia --from=$VALIDATOR_WALLET --chain-id=mamaki
```
Comment (adlerjohn) [inline docs/nodes/mamaki-testnet.md:126]: This doesn't actually look like a functional change? Revert the change, as a single long line doesn't display nicely with Docusaurus.

Comment (JB273) [reply docs/nodes/mamaki-testnet.md:126]: The line wit the `\` also is confusing for users as there are many spaces added when copy pasting this one...!
Rather have a working line opposed to one that users fid hard to interpret...
Up to you guys!

Comment (adlerjohn) [reply docs/nodes/mamaki-testnet.md:126]: You can literally copy-paste the lines with `\` in them. It's shell. Please revert.

Comment (YazzyYaz) [reply docs/nodes/mamaki-testnet.md:126]: Also, change `celesvaloper1q3v5cugc8cdpud87u4zwy0a74uxkk6u43cv6hd` to `celestiavaloper1q3v5cugc8cdpud87u4zwy0a74uxkk6u43cv6hd`
